Tests/SentryTests/Integrations/OutOfMemory/SentryOutOfMemoryTrackerTests.swift

```diff
@@ -207,7 +207,27 @@ class SentryOutOfMemoryTrackerTests: NotificationCenterTestCase {
         sut.start()
         assertNoOOMSent()
     }
-    
+
+    func testAppOOM_WithBreadcrumbs() {
+        let breadcrumb = Breadcrumb()
```

Comment:
`l`: You could use
https://github.com/getsentry/sentry-cocoa/blob/725723e10c8d2448514e4cd4cee4e71351922e9f/Tests/SentryTests/Protocol/TestData.swift#L17-L25